examples/hello/bpWriteReadKokkos/bpWriteReadKokkos.cpp

```diff
@@ -98,7 +98,7 @@ int BPRead(const std::string fname, const size_t N, int nSteps, const std::strin
 
 int main(int argc, char **argv)
 {
-    const std::vector<std::string> list_of_engines = {"BP4", "BP5"};
+    const std::vector<std::string> list_of_engines = {"BPFile"};
```

Comment:
Same comment here as for `bpWriteReadHip.cpp`

Reply:
I discussed that with @eisenhauer and he said that we should just remove it.